src/msix/AppxBundleManifest.cpp

```diff
@@ -141,8 +141,9 @@ namespace MSIX {
         const std::string& architecture,
         const std::string& publisher,
         std::vector<Bcp47Tag>& languages,
+		std::vector<std::string>& scales,
         APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE packageType):
-        m_factory(factory), m_fileName(name), m_size(size), m_offset(offset), m_languages(std::move(languages)), m_packageType(packageType)
+        m_factory(factory), m_fileName(name), m_size(size), m_offset(offset), m_languages(std::move(languages)), m_scales(std::move(scales)), m_packageType(packageType)

```

Comment:
>std::move [](start = 122, length = 9)

Don't move a &; if you want to move pass in a &&

Reply:
And I see that languages breaks that, but I didn't review that apparently...

---
In reply to: [288747023](https://github.com/Microsoft/msix-packaging/pull/146#discussion_r288747023) [](ancestors = 288747023)